clusters/build-clusters/01_cluster/openshift/api_audit_log/admin_vector_configmap.yaml

```diff
@@ -0,0 +1,25 @@
+kind: ConfigMap
+apiVersion: v1
+data:
+  vector.toml: |-
+    [sources.kube_apiserver_audit_logs]
+      type = "file"
+      include = ["/host/var/log/kube-apiserver/audit.log"]
+      ignore_older = 86400
+
+    [sources.openshift_apiserver_audit_logs]
+      type = "file"
+      include = ["/host/var/log/openshift-apiserver/audit.log"]
+      ignore_older = 86400
+
+    [sinks.aws_cloudwatch_logs]
```

Comment:
TODO: need to inject auth info after James provions the IAM user.

Ref: 
https://vector.dev/docs/reference/sinks/aws_cloudwatch_logs/#aws-authentication

Reply:
Done.